<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RDVTodo</class>
 <widget class="QDialog" name="RDVTodo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RDV à prendre</string>
  </property>
  <property name="accessibleName">
   <string>RDV à prendre</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="TypeRDV">
     <property name="text">
      <string>Type de RDV</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="DateLimite">
     <property name="text">
      <string>Date Limite</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="validateRDVtodo">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="typeRDV"/>
   </item>
   <item row="3" column="0">
    <widget class="QDateEdit" name="dateLimitEditRDV"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>validateRDVtodo</sender>
   <signal>accepted()</signal>
   <receiver>RDVTodo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>validateRDVtodo</sender>
   <signal>rejected()</signal>
   <receiver>RDVTodo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
